### PR TITLE
Reverse CLI command

### LINKED ISSUE #6: Feature request: HTML to HBML converter
Basically the reverse of the HBML parser in https://aydenh.ca/hbml/. I think it would be nice if one could simply paste some HTML code and have it transformed to HBML — it would definitely help conveying the appeal of this project. In a sense, it would have a role similar to the [HTML vs HBML examples](https://github.com/heyitsdoodler/hbml#html-vs-hbml-examples) section of the README, but it would be adaptable to the reader's own needs/preferences.

### PR DESCRIPTION
**Pull request by RosiePuddles**
As noted in #6, being able to convert HTML into HBML will lessen the barrier to entry to begin using HBML fo many people. This PR adds in a command to convert valid HTML into HBML.
The usage will be `hbml reverse [files]... (-o [output_prefix])`. The reterned HBML is formatted using the default linting options since this command is not intended for HBML linted, just making HTML form HBML. Any files given that do not exist or are invalid HTML are passed over and a warning is emitted. I've chosen this over an error to make the command do as much as it's been given, and allow the user to correct the incorrect files and re-run the command.
This will allow for futurre implementation of features noted in #6.

### CODE REVIEW

**Comment by waldyrious** [comment]
Nice! :star_struck: Thanks for taking this on :)

**Comment by heyitsdoodler** [comment]
I will add support for this when I update the online parser later today.

**Comment by RosiePuddles** [comment]
to add to this, this repo contains a CLI and JS module. When we add in features, we'll generally write the code for it and add it as a cli, but expose the main part of it as part of the module (`Parser` and `fullStringify` at the moment). This can then be used as part of a website or for other tools (express views for example). The CLI just generally comes first then everything else follows from exposed objects

**Comment by waldyrious** [comment]
Noted, thanks for the clarification! In any case, just for pedantry (:sweat_smile:), please change "close #6" in the PR description to "address #6" or some other wording that doesn't trigger automatic closing of #6 when this PR is merged, as it would then have to be reopened until the web converter is implemented :)

File: docs/src/commands/reverse.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Reverse
+
+The `hbml reverse` command allows you to convert HTML back into HBML.
+
+The command accepts a list of paths, an optional output prefix, and a verbose output option.
+
+It uses a HTML parser to tokenise the input which is then converted to HBML tokens and turned into linted HBML with the default linting options with one change to make elements with one child prefer the error operator over using brackets.
```
Comment (waldyrious) [inline docs/src/commands/reverse.md:7]:
```suggestion
It uses a HTML parser to tokenise the input which is then converted to HBML tokens and turned into linted HBML with the default linting options with one change to make elements with one child prefer the arrow operator over using brackets. 
```

**Comment by RosiePuddles** [comment]
once tests are written and they all pass, this will get merged

**Review by heyitsdoodler — APPROVED** [review]

**Pull request merged**